LogMonitor/src/LogMonitor/FileMonitor/Utilities.cpp

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+#include "pch.h"
+
+/**
+ * Warapper around Create Event API
+ * 
+ * @param bManualReset
+ * @param bInitialState
+ * 
+ * return event handle
+*/
+HANDLE CreateFileMonitorEvent(
+        _In_ BOOL bManualReset,
+        _In_ BOOL bInitialState
+) {
+    HANDLE event = CreateEvent(nullptr, bManualReset, bInitialState, nullptr);
```

Comment:
Their seem to be CreateEventA, CreateEventW, CreateEventExA, and CreateEventExW. Hopefully we are using the right one.

Reply:
I haven't changed this bit of code, I'll take a look at this in the second pass to figure out whether we will gain anything by changing the APIs